scripts/travis/build-all-in-one-image.sh

```diff
@@ -28,7 +28,7 @@ run_integration_test() {
 
```

Comment:
I think we should replace L11-12 with a check that the correct version of Node is in use and make it the responsibility of the caller. Also, executing `source` command when `set -x` is active creates a mess of the logs.

Reply:
sure

Reply:
Updated the PR with check for correct version